src/s390x/dasd.rs

```diff
@@ -33,18 +32,30 @@ use crate::runcmd;
 // IBM DASD Support
 /////////////////////////////////////////////////////////////////////////////
 
+const ECKD_DASD_BLOCKSIZE: u32 = 4096;
+
 #[derive(Debug)]
 struct Range {
     in_offset: u64,
     out_offset: u64,
     length: u64,
 }
 
-pub fn prepare_dasd(config: &InstallConfig) -> Result<()> {
-    low_level_format(&config.device)?;
-    if is_invalid(&config.device)? {
-        eprintln!("Disk {} is invalid, formatting", &config.device);
-        default_format(&config.device)?
+/// Returns default block size ony for unformatted ECKD DASD disk,
+/// otherwise None
```

Comment:
This doesn't really explain how the caller should use it.  How about something like

```suggestion
/// Returns expected sector size if this is a DASD we'll format later,
/// or None if the caller should use get_sector_size()
```